src/modules/storage/LedgerStorage.ts

```diff
@@ -471,13 +473,13 @@ export class LedgerStorage extends Storages
                     VALUES
                         (?, ?, ?, ?, ?, ?, ?)`,
                     [
-                        height,
+                        height.toString(),
                         tx_idx,
                         out_idx,
                         hash.toBinary(Endian.Little),
                         utxo_key.toBinary(Endian.Little),
                         output.address.toString(),
-                        output.value
+                        Utils.UInt64ToString(output.value)
```

Comment:
The Amount type has been treated as String.
I think it's good.
It would be good to provide an arithmetic function for the Amount type later.